pages/jobs.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <section class="section">
+    <div class="container">
+      <div v-for="job in jobPostings" :key="job.headline" class="mb-5">
+        <h2 class="heading heading-is-5 is-size-2-mobile text-bold">
+          {{ job.headline }}
+          <span class="text__stroked heading heading-is-7 is-size-1-mobile">{{ job.subheadline }}</span>
+        </h2>
+        <p class="max-w-600">{{ job.description }}</p>
+        <b-button
+          tag="a"
+          :href="job.url"
+          target="_blank"
+          rel="noopener noreferrer"
+          type="my-5 is-primary"
+        >
+          {{ $t('Learn More') }}
+        </b-button>
+      </div>
+   </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+
+@Component({
+  name: 'JobsPage',
+  layout: 'full-width-layout'
+})
```

Comment:
I didn't know you can put layout here, but `full-width-layout` was for landing page. 
Default layout might be good but you'll need to remove `<section class="section">` wrapper (see Faq)

Reply:
I see, just removed layout and added `/jobs` to `<Footer>` 👍 
<img width="585" alt="Screenshot 2022-01-05 at 09 37 43" src="https://user-images.githubusercontent.com/17953978/148187011-5f782e58-630c-4228-91fa-d3b25b18a9f2.png">


yes as far is i know you can actually put every Nuxt-Hook into the `@Component` part such as `head()` and `asyncData`